utils/ChromiumDownloader.js

```diff
@@ -97,7 +97,7 @@ module.exports = {
       fs.mkdirSync(DOWNLOADS_FOLDER);
     return downloadFile(url, zipPath, progressCallback)
         .then(() => extractZip(zipPath, folderPath))
-        .catch(err => err)
+        .catch(err => { throw err; })
         .then(() => {
           if (fs.existsSync(zipPath))
```

Comment:
Unfortunately, there's no "finally" in promise chains, so I'd fake it like this:
```js
  .catch(err => err)
  .then(err => {
    if (fs.existsSync(zipPath))
      fs.unlinkSync(zipPath);
    if (err)
      throw err;
  });
```
This way we'll throw an error and cleanup zip if any

Reply:
Much better love it. I've updated my code to do that.